newsroom/companies/__init__.py

```diff
@@ -1,39 +1,32 @@
+import superdesk
+
 from flask_babel import lazy_gettext
 
-import superdesk
+from newsroom.companies.companies_async.types import CompanyResource
```

Comment:
`CompanyResource` is exposed by the top level module, so you can import by using:

```python
from newsroom.companies import CompanyResource
```

This is a good example of a module having a defined interface of what it exposes. It might be good if we try and follow that with the modules, so we're not importing from too deep within the modules. `from newsroom.<x> import <y>`.

Reply:
yeah, I think this was just auto import from the editor. I'll fix it